staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -386,7 +391,62 @@ func (s *store) GetToList(ctx context.Context, key string, resourceVersion strin
 		return err
 	}
 	// update version with cluster level revision
-	return s.versioner.UpdateList(listObj, uint64(getResp.Header.Revision))
+	return s.versioner.UpdateList(listObj, uint64(getResp.Header.Revision), "")
+}
+
+// continueToken is a simple structured object for encoding the state of a continue token.
+// TODO: if we change the version of the encoded from, we can't start encoding the new version
+// until all other servers are upgraded (i.e. we need to support rolling schema)
+// TODO: be very careful about changing this
```

Comment:
Should this be a TODO? It sounds like a real comment.

Reply:
Yeah, will update.